### PR TITLE
reorganized and updated strings for german translation

### PR DESCRIPTION
**Pull request by GuepardoApps**
added missing translations
reordered some values for better comparison

### CODE REVIEW

File: mobile/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -21,7 +27,8 @@
     <string name="settings_openhab_sslclientcert">SSL-Client-Zertifikat</string>
     <string name="settings_openhab_sslclientcert_howto">Erläuterungen zum Client-Zertifikat</string>
     <string name="settings_openhab_sslclientcert_howto_summary">Für Hinweise und Erläuterungen zur SSL-Authentifizierung mittels Client-Zertifikat hier tippen</string>
-    <string name="settings_openhab_screentimeroff">Ruhezustand deaktivieren</string>
+    
+	<string name="settings_openhab_screentimeroff">Ruhezustand deaktivieren</string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-de/strings.xml]: Wrong indention

File: mobile/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -33,22 +40,52 @@
     <string name="settings_openhab_fullscreen">Vollbild</string>
     <string name="settings_openhab_fullscreen_summary">Geräte-Statusleiste während der Verwendung der App ausblenden</string>
     <string name="settings_openhab_icon_format">Symbol-Format</string>
-    <string name="settings_ringtone">Klingelton</string>
+    
+	
+	
+	
+	<string name="settings_ringtone">Klingelton</string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-de/strings.xml]: Wrong indention

File: mobile/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -67,15 +104,21 @@
     <string name="action_settings">Einstellungen</string>
     <string name="nfc_dialog_title">Aktion des NFC-Tags</string>
     <string name="info_not_set">Nicht gesetzt</string>
-    <!-- Strings for MemorizingTrustManager activity -->
+    
+	<!-- Strings for MemorizingTrustManager activity -->
```
Comment (mueller-ma) [inline mobile/src/main/res/values-de/strings.xml]: Wrong indention

File: mobile/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -67,15 +104,21 @@
...
+    <string name="mtm_notification">Verifiziere Zertifikat</string>
     <string name="settings_openhab_none">Keins</string>
     <!-- theme -->
-    <string name="theme_name_light">Hell</string>
+    
+	
+	
+	
+	
+	<string name="theme_name_light">Hell</string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-de/strings.xml]: Wrong indention

File: mobile/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -85,6 +128,7 @@
     <string name="nfc_action_on">An</string>
     <string name="nfc_action_off">Aus</string>
     <string name="nfc_action_toggle">Umschalten</string>
+    <string name="nfc_action_to_sitemap_page">Navigiere zur Sitemap-Seite</string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-de/strings.xml]: "Navigiere zur Sitemap"?

File: mobile/src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -94,42 +138,21 @@
     <!-- About -->
     <string name="about_title">Über</string>
     <string name="about_license_title">Lizenz</string>
-    <string name="about_links">Links</string>
+    
+	<string name="about_links">Links</string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-de/strings.xml]: Wrong indention

File: mobile/src/main/res/values-de/strings.xml [hunk 7]
```diff
@@ -94,42 +138,21 @@
...
-    <string name="error_unable_to_resolve_hostname">Fehler beim Auflösen des Hostnamens</string>
+    <string name="unknown">Unbekannt</string>
+    <string name="error_network_type_unsupported">Netzwerktyp (%s) wird nicht unterstützt</string>
     <string name="set">Setzen</string>
     <string name="cancel">Abbrechen</string>
-    <string name="dlg_notification_title">Benachrichtigung</string>
-    <string name="settings_no_default_sitemap">Keine Standard-Sitemap festgelegt</string>
-    <string name="settings_clear_default_sitemap">Standard-Sitemap zurücksetzen</string>
-    <string name="settings_current_default_sitemap">Aktuelle Standard-Sitemap: %1$s</string>
+	<string name="app_intro">Intro</string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-de/strings.xml]: Wrong indention

File: mobile/src/main/res/values-de/strings.xml [hunk 8]
```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
     <!-- General app strings -->
+
```
Comment (FlorianSW) [inline mobile/src/main/res/values-de/strings.xml]: why adding so much empty lines? :/

Comment (GuepardoApps) [reply mobile/src/main/res/values-de/strings.xml]: empty lines are placeholders for not translatable values. It's easier to compare the default and the german strings.xml

**Comment by mueller-ma** [comment]
@GuepardoApps Its very hard to see what string you really changed. Can you use [crowdin](https://crowdin.com/translate/openhab-android/7/en-de) to update the translation?

File: mobile/src/main/res/values-de/strings.xml [hunk 9]
```diff
@@ -85,6 +128,7 @@
     <string name="nfc_action_on">An</string>
     <string name="nfc_action_off">Aus</string>
     <string name="nfc_action_toggle">Umschalten</string>
+    <string name="nfc_action_to_sitemap_page">Navigiere zur Sitemap</string>
```
Comment (maniac103) [inline mobile/src/main/res/values-de/strings.xml]: Zu Sitemap navigieren

File: mobile/src/main/res/values-de/strings.xml [hunk 10]
```diff
@@ -33,22 +40,52 @@
...
+
+
+
     <string name="settings_ringtone">Klingelton</string>
+    <string name="settings_clear_default_sitemap">Standard-Sitemap zurücksetzen</string>
+    <string name="settings_no_default_sitemap">Keine Standard-Sitemap festgelegt</string>
+    <string name="settings_current_default_sitemap">Aktuelle Standard-Sitemap: %1$s</string>
+    <string name="settings_openhab_sslsettings">Verschlüsselungseinstellungen</string>
+    <string name="settings_openhab_alt_connection">Einstellungen für Remote-Server</string>
+    <string name="settings_openhab_connection">Einstellungen für lokale Server</string>
```
Comment (maniac103) [inline mobile/src/main/res/values-de/strings.xml]: Einstellungen fûr lokalen Server

**Pull request closed without merging**